packages/proto-loader/README.md

```diff
@@ -51,3 +52,66 @@ const options = {
   oneofs: true
 }
 ```
+
+## Generating TypeScript types
+
+The `proto-loader-gen-types` script distributed with this package can be used to generate TypeScript type information for the objects loaded at runtime. More information about how to use it can be found in [the *@grpc/proto-loader TypeScript Type Generator CLI Tool* proposal document](https://github.com/grpc/proposal/blob/master/L70-node-proto-loader-type-generator.md). The arguments mostly match the `load` function's options; the full usage information is as follows:
+
+```console
+proto-loader-gen-types.js [options] filenames...
+
+Options:
+  --help             Show help                                         [boolean]
+  --version          Show version number                               [boolean]
+  --keepCase         Preserve the case of field names                  [boolean]
```

Comment:
does this work? I've tried passing `--keepCase false, `--keepCase true` and `--no-keepCase` without it affecting output

(what I want is to _just_ generate the camel cased ones)

Reply:
It's just a flag. You pass the option to get that behavior, and omit it to get the other behavior.

Reply:
Right, but whatever value I pass as this flag the generated code is identical. If it doesn't change anything it should be removed, no?

Reply:
It doesn't have a value. You either pass it or you don't. So, if you want the original field name case from the proto files, you pass `--keepCase`. And if you want it to force camel case names, you don't pass that flag at all.

Reply:
`yargs` should allow you to pass `false` to negate boolean flags.

However, not passing it at all also makes no difference in the generated code.

Reply:
It works for me. I tried modifying the `generate-golden` script introduced in this PR to remove that argument, and the field names generated in camel case instead of the original casing, as expected. I want to note that this only affects field names that are written in a format other than camel case, such as snake case with underscores. It won't affect message or service names, or field names that are already just one word.

Reply:
Aha, gotcha! Yeah, that's it then, I was looking at message names, all field names are already came case in our case 👍